Formula/assh.rb

```diff
@@ -18,7 +17,7 @@ class Assh < Formula
   def install
     ENV["GOPATH"] = buildpath
     (buildpath/"src/github.com/moul/advanced-ssh-config").install Dir["*"]
-    cd "src/github.com/moul/advanced-ssh-config/cmd/assh" do
+    cd "src/github.com/moul/advanced-ssh-config" do
       system "go", "build", "-o", bin/"assh"
       prefix.install_metafiles
     end
```

Comment:
```suggestion
    system "go", "build", "-ldflags", "-s -w", "-trimpath", "-o", bin/"assh"
    prefix.install_metafiles
```
Since we're working on GOPATH deprecation: https://github.com/Homebrew/homebrew-core/issues/47627